google-cloud-spanner/src/test/java/com/google/cloud/spanner/InstanceAdminClientImplTest.java

```diff
@@ -408,14 +531,53 @@ public void testUpdateInstanceWithNodeCountAndProcessingUnits() throws Exception
     when(rpc.updateInstance(
             instance,
             FieldMask.newBuilder()
-                .addAllPaths(Arrays.asList("display_name", "node_count", "labels"))
+                .addAllPaths(
+                    Arrays.asList("display_name", "autoscaling_config", "node_count", "labels"))
+                .build()))
+        .thenReturn(rawOperationFuture);
+    InstanceInfo instanceInfo =
+        InstanceInfo.newBuilder(InstanceId.of(INSTANCE_NAME))
+            .setInstanceConfigId(InstanceConfigId.of(CONFIG_NAME))
+            .setNodeCount(3)
+            .setProcessingUnits(3000)
+            .build();
+    OperationFuture<Instance, UpdateInstanceMetadata> operationWithFieldMask =
+        client.updateInstance(instanceInfo);
+    assertTrue(operationWithFieldMask.isDone());
+    assertEquals(INSTANCE_NAME, operationWithFieldMask.get().getId().getName());
```

Comment:
Could you add more assertions on fields being updated?

Reply:
Same above